invenio_app_ils/document_requests/loaders/jsonschemas/document_request_accept.py

```diff
@@ -20,3 +22,5 @@ class Meta:
         unknown = EXCLUDE
 
     document_pid = fields.Str()
+    physical_item_provider = fields.Nested(PhysicalItemProviderSchema)
+    state = fields.Str()
```

Comment:
Should any of these be `required`?

Reply:
Non of them is required because the endpoint will accept requests which contain some of them